rts/Game/UnsyncedGameCommands.cpp

```diff
@@ -3833,85 +3875,29 @@ class CommandHelpActionExecutor : public IUnsyncedActionExecutor {
 } // namespace (unnamed)
 
 
-
-
-
-// TODO CGame stuff in UnsyncedGameCommands: refactor (or move)
-bool CGame::ActionReleased(const Action& action)
+bool UnsyncedGameCommands::ActionPressed(const Action& action, bool isRepeat)
 {
-	switch (hashString(action.command.c_str())) {
-		case hashString("drawinmap"): {
-			inMapDrawer->SetDrawMode(false);
-		} break;
-
-		case hashString("moveforward"): {
```

Comment:
Where's the handling of `move<direction>` releases?

There are some options:

- Implement `"move<direction>"` in a similar manner to `"groupN"`
- Same as above but we'd also have a single executor `movecamera <direction>` to simplify stuff, and relay from the terribly named action above to the simplified here. If too much for the scope of the PR feel free to ignore this option.